### DISCUSSION TITLE
What theme are you using?

### DISCUSSION THREAD

**Discussion by Corey-Keller** [ggandor/lightspeed.nvim#54] [Q&A] [answered]
I'm wondering what theme your using in the screenshots (and what theme(s) you're personally using currently if different)

EDIT: Sorry ggandor. I always forget about `Discussions` being a thing.

**Comment by ggandor** [comment, marked as answer]
The theme used for the screenshots is [bogster](https://github.com/vv9k/bogster). Among the dark themes, one of my favourites currently is [nightfox](https://github.com/EdenEast/nightfox.nvim/), but [nord](https://github.com/shaunsingh/nord.nvim) and [tokyonight](https://github.com/folke/tokyonight.nvim) are also great. Some light themes I use a lot are [vim-colors-xcode](https://github.com/arzg/vim-colors-xcode), [github-light](https://github.com/romgrk/github-light.vim), and [space-vim-theme](https://github.com/liuchengxu/space-vim-theme). In addition, [zenbones](https://github.com/mcchrish/zenbones.nvim) is a particularly nice new low-saturation theme.

> I always forget about Discussions being a thing.

No problem, me too 😊
